main.cpp

```diff
@@ -72,45 +85,83 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 } 
 
 /*
- 1)
+ 1)Rent a bike
  */
+bool rentBike(double rentDate, int amountOfMoney = 10)
+{
+  ignoreUnused(rentDate, amountOfMoney);
 
+  return{};
+}
 /*
- 2)
+ 2)Play sound
  */
-
+void playSound(float wavFile, float fs = 44100.0f)
+{
+ ignoreUnused(wavFile, fs);
+}
```

Comment:
Please fix the indentation so it conforms with the coding standard.  You'll find the course coding standard in the Readme.md.
- [ ] revise